waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -173,23 +171,23 @@ warnProjectDiffersFromDb projectRootDir = do
         "Wasp was unable to verify your database is up to date."
           <> " Running `wasp db migrate-dev` may fix this and will provide more info."
 
-genPrismaClients :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO (Maybe GeneratorError)
-genPrismaClients spec projectRootDir =
+generatePrismaClient :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO (Maybe GeneratorError)
+generatePrismaClient spec projectRootDir =
```

Comment:
Because it used to be "clients" (plural) and would generate two of them, the `gen` -> `generate` part overshadowed the more important change :D

> So why did you comment this here? I also wonder, should we have this comment somewhere in the actual code in the relevant place?

I don't think so. Once we merge this, the fact that we once had two clients as a workaround becomes unimportant.

Reply:
Aha I didn't notice the change in plurality!

I was referring to the comment that it generates a single Prisma client in `node_modules/.prisma`, I thought that was worth documenting somewhere, not the fact that there were 2 clients before.

Reply:
Hm, I'd skip documenting that one as well.
Generating the client in `node_modules` is the default and expected behavior from Prisma. Our previous approach was also a very custom workaround.

Reply:
It is, but still it doesn't mean it is very intuitive. I would mention it shortly in the place that actually performs that install.

Reply:
Just to check, since there is no :rocket: -> is this done?